### PR TITLE
android/*: add French translation

### PR DESCRIPTION
**Pull request by Nico385412**
- [X] The page (if new), does not already exist in the repository.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

 - **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.fr/android/am.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# am
+
+> Manager d'activité Android.
+> Plus d'informations : <https://developer.android.com/studio/command-line/adb#am>.
+
+- Commence une activité spécifique:
```
Comment (blueskyson) [inline pages.fr/android/am.md:6]: Thank you for contribution. Every page in `page.fr` puts spaces in front of colons.
```suggestion
- Commence une activité spécifique :
```

Comment (Nico385412) [reply pages.fr/android/am.md]: done ✅

File: pages.fr/android/dumpsys.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# dumpsys
+
+> Fourni des informations sur les services du système Android.
+> Cette commande peut être utilisé uniquement depuis `adb shell`.
+> Plus d'informations : <https://developer.android.com/studio/command-line/dumpsys>.
+
+- Récupère un diagnostic pour chaque service système:
```
Comment (navarroaxel) [inline pages.fr/android/dumpsys.md:7]:
```suggestion
- Récupère un diagnostic pour chaque service système :
```

File: pages.fr/android/getprop.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# getprop
+
+> Affiche les informations des propriétés système Android.
+> Plus d'informations : <https://manned.org/getprop>.
+
+- Affiche les informations des propriétés système Android:
```
Comment (navarroaxel) [inline pages.fr/android/getprop.md:6]:
```suggestion
- Affiche les informations des propriétés système Android :
```

File: pages.fr/android/getprop.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+`getprop`
+
+- Affiche les informations d'une propriété spécifique:
```
Comment (navarroaxel) [inline pages.fr/android/getprop.md:10]:
```suggestion
- Affiche les informations d'une propriété spécifique :
```

Comment (Nico385412) [reply pages.fr/android/getprop.md]: done 👍

File: pages.fr/android/dumpsys.md (hunk 2 above)
Comment (Nico385412) [reply pages.fr/android/dumpsys.md]: done 👍

File: pages.fr/android/getprop.md (hunk 3 above)
Comment (Nico385412) [reply pages.fr/android/getprop.md]: done 👍

**Review by navarroaxel — APPROVED** [review]

**Review by blueskyson — APPROVED** [review]

**Pull request merged**